lib/routes/itc/collection.ts

```diff
@@ -0,0 +1,67 @@
+import { Data, Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+/**
+ * OpenGithub - Github开源项目精选
+ *  colType：0 | 1 | 2 | 3
+ */
+
+enum COL_TYPE {
+    HANDPICK,
+    SPECIAL,
+    WEEKLY,
+    MONTHLY,
+}
+
+const RESULT_DESC_MAP = {
+    [COL_TYPE.HANDPICK]: 'OpenGithub - Github开源项目精选 - 精选文章',
+    [COL_TYPE.SPECIAL]: 'OpenGithub - Github开源项目精选 - 专栏',
+    [COL_TYPE.WEEKLY]: 'OpenGithub - Github开源项目精选 - 周刊',
+    [COL_TYPE.MONTHLY]: 'OpenGithub - Github开源项目精选 - 月刊',
+};
+
+export const route: Route = {
+    path: '/collection/:colType',
+    categories: ['blog'],
+    example: '/itc/collection/1',
+    radar: [
+        {
+            source: ['open.itc.cn/'],
+        },
+    ],
+    name: '合集',
+    maintainers: ['cnkmmk'],
+    handler,
+};
+
+async function handler(ctx) {
+    const url = 'https://open.itc.cn';
+
+    const colType = ctx.req.param('colType');
+
+    const result = {
+        title: '',
+        link: url,
+        description: RESULT_DESC_MAP[colType] ?? 'NULL',
+        item: [] as Data[],
```

Comment:
The type of returned object is `Data`. The type of `item` in the returned object is [DataItem](https://github.com/DIYgod/RSSHub/blob/a51cc049bf67c24216953449f1a2d500fe7f0367/lib/types.ts#L31).